### DISCUSSION TITLE
focus-visible not working on Safari

### DISCUSSION THREAD

**Discussion by sidgoel85** [tailwindlabs/tailwindcss#4932] [Help]
### What version of Tailwind CSS are you using?

2.1.2

### What build tool (or framework if it abstracts the build tool) are you using?

webpack with craco

### What version of Node.js are you using?

12.2.0

### What browser are you using?

Chrome/Safari/Firefox/Edge

### What operating system are you using?

Windows

### Reproduction repository

https://github.com/sidgoel85/focus_visible

### Describe your issue

Hi,
I'm trying to get focus-visible to work in safari but it just refuses to work as expected. I've followed all the steps as outlined here : https://tailwindcss.com/docs/hover-focus-and-other-states#focus-visible with no success.

Note- The same changes work in Chrome/Firefox/Edge.

**Comment by bradlc** [comment]
Hey @sidgoel85. It looks like you are already using `postcss-focus-visible`, but you need to be using the [`focus-visible` JavaScript polyfill](https://github.com/WICG/focus-visible) as well. Either include the script from a CDN:

```html
<script src="https://unpkg.com/focus-visible@5.2.0/dist/focus-visible.min.js"></script>
```

Or install it via `npm` and import it:

```
npm install focus-visible
```

```js
// main.js
import 'focus-visible'
```

**Comment by sidgoel85** [comment]
Hey @bradlc - We've already installed the necessary plugins like above and it still doesn't work. Please note that same steps are working in other browsers.

**Comment by bradlc** [comment]
I see. In that case can you update your reproduction repository so that the issue is observable? Currently it contains an invalid `package.json` file and no way to actually reproduce the issue. I'm going to move this to discussions as well since this isn't an issue with Tailwind 👍

**Reply by sidgoel85** [reply]
Thanks, Updated package.json with the complete details here : https://github.com/sidgoel85/focus_visible/blob/main/package.json

We've been kind of stuck on this issue for sometime now and will really appreciate any help.

**Reply by sidgoel85** [reply]
Hi All, Any help here please?

**Comment by sarayourfriend** [comment]
@sidgoel85 If you still haven't managed to get it to work, we just ran into this issue on Openverse and were able to make the polyfill and PostCSS plugin work together: https://github.com/WordPress/openverse-frontend/pull/724

We did have to change around some of our classes to fix issues that only appeared in Safari with the polyfill, but it does work perfectly now.

Good luck!
